packages/veritone-widgets/test/helpers/utils.js

```diff
@@ -0,0 +1,20 @@
+import _ from 'lodash';
+// Removes all functions in an object so you can test
+// against objects without errors
+export const removeFunctions = (obj) => {
```

Comment:
since you're writing this in an immutable style (good), call it `withoutFunctions`, since `remove` verb implies youre gonna mutate the input. (can look at lodash for examples in this vein, ie _.without vs _.remove)